src/ethereum/london/vm/gas.py

```diff
@@ -59,6 +62,36 @@
 GAS_WARM_ACCESS = Uint(100)
 
 
+@dataclass
+class ExtendMemory:
+    """
+    Define the parameters for memory extension in opcodes
+
+    `cost`: `ethereum.base_types.Uint`
+        The gas required to perform the extension
+    `size`: `ethereum.base_types.Uint`
+        The size by which the memory will be extended
+    """
+
+    cost: Uint
+    size: Uint
```

Comment:
I see that the other docstrings use the former syntax. If we have to change the syntax, I guess it will be a good idea to create a new issue and change the syntax for all the docstrings.

Reply:
Opened a new issue #671